### PR TITLE
docs: fixed sql error of the last example in having_clause

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs
* **What is the current behavior?** (You can also link to an open issue here)
when running the sql command of the last example (**line 44**), below error msg appeared
**Error: no tablet available for sqlHaving clause is not in GROUP BY clause and contains nonaggregated column 'db1.t2.COL2' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by**
* **What is the new behavior (if this is a feature change)?**
change the having clause of  this sql command and it works
other syntax errors have also been fixed, like **line 3**

### CODE REVIEW

File: docs/zh/reference/sql/dql/HAVING_CLAUSE.md [hunk 1]
```diff
@@ -16,31 +16,28 @@ SELECT select_expr [,select_expr...] FROM ... GROUP BY ... HAVING having_conditi
...
 
 ## 边界说明
-
-| SELECT语句元素 | 状态          | 说明                                                         |
-| :------------- | ------------- | :----------------------------------------------------------- |
-| HAVING Clause  | Online 不支持 | Having 子句与 Where 子句作用类似，Having 子句可以让过滤 GroupBy 后的各种数据，Where 子句用于在聚合前过滤记录。 |
+在单机版中，所有执行模式均支持`HAVING`。集群版各[执行模式](https://openmldb.ai/docs/zh/main/tutorial/modes.html)的支持情况如下。
+| SELECT语句元素 | 状态          | 说明                                                                  |
+| :------------- | ------------- |:--------------------------------------------------------------------|
+| HAVING Clause  | 仅支持`Offline`模式，`Online`和`Request`模式均不支持| Having 子句与 Where 子句作用类似.Having 子句过滤 GroupBy 后的各种数据，Where 子句在聚合前进行过滤。|
```
Comment (vagetablechicken) [inline docs/zh/reference/sql/dql/HAVING_CLAUSE.md]: 执行模式batch/reqeust和集群模式online/offline是不同的，可以说有offline batch（offline没有支持request），online batch（CLI里命令全是batch）， online request（通过执行deploy和sp）这几种

Comment (michelle-qinqin) [reply docs/zh/reference/sql/dql/HAVING_CLAUSE.md]: 我明白了。
但是，这个表是[select概述](https://openmldb.ai/docs/zh/main/reference/sql/dql/SELECT_STATEMENT.html)的子表。
select概述里的表似乎是为了和[集群版使用流程及执行模式](https://openmldb.ai/docs/zh/main/tutorial/modes.html)里介绍的三种执行模式相对应？

Comment (vagetablechicken) [reply docs/zh/reference/sql/dql/HAVING_CLAUSE.md]: 行，那就对应来

**Review by vagetablechicken — APPROVED** [review]

**Review by aceforeverd — APPROVED** [review]

File: docs/zh/reference/sql/dql/HAVING_CLAUSE.md (hunk 1 above)
Comment (lumianph) [inline docs/zh/reference/sql/dql/HAVING_CLAUSE.md]: let's change them to the consist names 离线模式，在线预览模式，在线请求模式

File: docs/zh/reference/sql/dql/HAVING_CLAUSE.md [hunk 2]
```diff
@@ -16,31 +16,28 @@ SELECT select_expr [,select_expr...] FROM ... GROUP BY ... HAVING having_conditi
...
 
 ## Example
 
 ### 1. 分组后按聚合结果过滤
 
 ```SQL
--- desc: 分组后聚合过滤
-SELECT COL1, SUM(COL2), AVG(COL2) FROM t1 group by COL1 HAVING SUM(COL2) > 1000;
+>SELECT COL1, SUM(COL2), AVG(COL2) FROM t1 group by COL1 HAVING SUM(COL2) > 1000;
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/HAVING_CLAUSE.md]: remove `>`

File: docs/zh/reference/sql/dql/HAVING_CLAUSE.md [hunk 3]
```diff
@@ -16,31 +16,28 @@ SELECT select_expr [,select_expr...] FROM ... GROUP BY ... HAVING having_conditi
...
 ```
 
 ### 2. 两列分组后按聚合结果过滤
 
 ```sql
--- desc: 分组后聚合过滤
-SELECT COL1, SUM(COL2), AVG(COL2) FROM t1 group by COL1, COL0 HAVING SUM(COL2) > 1000;
+>SELECT COL1, SUM(COL2), AVG(COL2) FROM t1 group by COL1, COL0 HAVING SUM(COL2) > 1000;
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/HAVING_CLAUSE.md]: remove `>`

File: docs/zh/reference/sql/dql/HAVING_CLAUSE.md [hunk 4]
```diff
@@ -16,31 +16,28 @@ SELECT select_expr [,select_expr...] FROM ... GROUP BY ... HAVING having_conditi
...
 ```
 
 ### 3. 分组后按分组列过滤
 
 ```sql
--- desc: 分组后聚合过滤
-SELECT COL1, SUM(COL2), AVG(COL2) FROM t1 group by COL1 HAVING COL2 > 1000;
+>SELECT COL1, SUM(COL2), AVG(COL2) FROM t1 group by COL1 HAVING COL1 ='a';
```
Comment (lumianph) [inline docs/zh/reference/sql/dql/HAVING_CLAUSE.md]: remove `>`

File: docs/zh/reference/sql/dql/HAVING_CLAUSE.md (hunk 2 above)
Comment (lumianph) [reply docs/zh/reference/sql/dql/HAVING_CLAUSE.md]: not necessary for this doc

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/zh/reference/sql/dql/HAVING_CLAUSE.md (hunk 1 above)
Comment (michelle-qinqin) [reply docs/zh/reference/sql/dql/HAVING_CLAUSE.md]: the corrected version containing following comments has been commit ~

**Review by lumianph — APPROVED** [review]

**Pull request merged**